code/obj/critter/bee.dm

```diff
@@ -1440,7 +598,794 @@
+				src.set_density(initial(src.density))
+				src.on_revive()
+				src.visible_message("<span class='alert'>[src] seems to rise from the dead!</span>")
+
+/obj/critter/domestic_bee/zombee/lich // sprite by mageziya, it's silly but cute imo
+	name = "lich-bee"
+	icon_state = "lichbee-wings"
+	icon_body = "lichbee"
+	sleeping_icon_state = "lichbee-sleep"
+	honey_color = rgb(25, 55, 25)
+	cant_take_hat = 1
+	generic = 0
+
+/obj/critter/domestic_bee/small
+	icon_state = "lilbee-wings"
+	icon_body = "lilbee"
+	sleeping_icon_state = "lilbee-sleep"
+	icon_antenna = "antenna-lilbee"
+	hat_y_offset = -1
+	hat_x_offset_left = 1
+	hat_x_offset_right = 6
+
+/obj/critter/domestic_bee/sea // c b
+	name = "greater domestic sea-bee"
+	desc = "Genetically engineered for extreme size and indistinct segmentation and bred for docility, the greater domestic sea-bee is increasingly popular among ocean traders and science-types."
+	icon_state = "seabee-wings"
+	icon_body = "seabee"
+	sleeping_icon_state = "seabee-sleep"
+
+/obj/critter/domestic_bee/sonic
+	name = "sonic bee"
+	desc = "OH GOD IT IS BACK, WE WERE SURE WE REMOVED IT FROM THE CODEBASE BUT IT KEEPS COMING BACK OH GOD"
+	icon_body = "sonicbee"
+	icon_state = "sonicbee-wings"
+	sleeping_icon_state = "sonicbee-sleep"
+
+/obj/critter/domestic_bee/ascbee
+	name = "ASCBee"
+	desc = "This bee looks rather... old school."
+	icon_body = "ascbee"
+	icon_state = "ascbee-wings"
+	sleeping_icon_state = "ascbee-sleep"
+	angertext = "beeps aggressively at"
+	honey_color = rgb(0, 255, 0)
+
+	attack_hand(mob/user as mob)
+		if(src.alive && user.a_intent=="help")
+			src.visible_message("<span class='emote'><b>[user]</b> [pick("pets","hugs","snuggles","cuddles")] [src]!</span>")
+			if(prob(15))
+				for(var/mob/O in hearers(src, null))
+					O.show_message("<span class='emote'><b>[src]</b> beeps[prob(50) ? " in a comforted manner, and gives [user] the ASCII" : ""].</span>",2)
+			return
+		else
+			..()
+
+/obj/critter/domestic_bee/rgbee
+	name = "chromatic space bee"
+	desc = "Genetically engineered for extreme size and indistinct segmentation and bred for docility, the greater domestic space-bee is increasingly popular among space traders and science-types. This one's really colorful!"
+	icon_body = "rgbee"
+	icon_state = "rgbee-wings"
+	sleeping_icon_state = "rgbee-sleep"
+	icon_antenna = "antenna-empty"
+
+/obj/critter/domestic_bee/rgbee/do_hug(mob/user as mob)
+	user.visible_message("<span class='notice'>[src] hugs [user] back!</span>", "<span class='notice'>[src] hugs you back! You feel colorful inside!</span>")
+	if (user.reagents)
+		user.reagents.add_reagent("hugs", 3)
+		user.reagents.add_reagent("colors", 6)
+
+/obj/critter/domestic_bee/rgbee/do_reagentStuff(mob/M)
+	if (M.reagents.get_reagent_amount("methamphetamine") < 5)
+		M.reagents.add_reagent("methamphetamine", 1)
+	if (M.reagents.get_reagent_amount("colors") < 20)
+		M.reagents.add_reagent("colors", 5)
+
+/obj/critter/domestic_bee/lsbee
```

Comment:
oof i accidentally deleted the lsbee reagentstuff and do_hug, can you fix?

Reply:
Uh yeah one sec.

Reply:
Done.